docs/SOPs/archiving_SOP.md

```diff
@@ -99,6 +99,8 @@ Once a submission is ready in ingest (`Archiving` status after hitting submit),
 
 ## Step 2 of 3 - Archiving Files to DSP
 
+_Note: When direct archiving is fully functional, the instructions at the end of this step on __Uploading files directly to ENA__ should be followed instead of the following._
+
```

Comment:
I am still not completely clear on this. Does this mean that direct archiving, which is simpler, is not yet functional, i.e. we cannot follow that step yet? Is there a way to make it more clear which steps cannot be run currently, for example, by highlighting that section in a different colour?